factory-containers/build.sh

```diff
@@ -84,9 +89,19 @@ for x in $IMAGES ; do
 		auth=1
 	fi
 
-	status Building docker image $x for $ARCH
 	cd $x
+<<<<<<< HEAD
```

Comment:
When I fixed these merge conflicts, I ended up committing them to the next patch. Let me know if you want me to fix this here.